### PR TITLE
Logging Moderation Actions to database and `moderation_actions` command

### PR DESCRIPTION
**Pull request by WillFJS**

### CODE REVIEW

File: src/bot/commands/warn.js [hunk 1]
```diff
@@ -63,5 +67,7 @@ module.exports = {
 
         /* message the member in the server */
         await message.channel.send(moderation_message_contents).catch(console.warn);
+
+        await logModerationActionToDatabase(member.id, 'warn', Date.now(), message.member.id, reason);
```
Comment (MidSpike) [inline src/bot/commands/warn.js:71]: `ModerationActionType` should be capitalized

**Review by MidSpike — CHANGES_REQUESTED** [review]

**Review by MidSpike — CHANGES_REQUESTED** [review]
@WillFJS You need to change how you are using `logModerationActionToDatabase()` in the commands

The function accepts 2 objects

```
logModerationActionToDatabase({ discord_user_id }, { type, epoch, reason, staff_member_id });
```

**Review by MidSpike — CHANGES_REQUESTED** [review]
Please swap order to be:
```diff
  type
  epoch
- staff_member_id
  reason
+ staff_member_id
```

**Comment by MidSpike** [comment]
@WillFJS please test this when you get a chance

**Comment by MidSpike** [comment]
Everything looks good so far, I just need to finish the `moderation_actions` (`ma`) command

**Comment by MidSpike** [comment]
@WillFJS please review

**Comment by MidSpike** [comment]
Can confirm that everything is working as intended now

**Review by MidSpike — APPROVED** [review]

**Review by rossloika — APPROVED** [review]

**Review by DJHARDY2K21 — APPROVED** [review]

**Pull request merged**